### ISSUE TITLE
2.7.0-rc2 pre-release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2523]
- #2505 
- #2498
- #2500 
- #2513 
- #2496
- #2511
- #2507 
- #2517
- #2508
- #2501
- #2510
- #2522
- #2514 
- #2448
- #2520

## Pick list
```
git cherry-pick 8bf44844adab955866d1a54a8db9c37a3e4570a7
git cherry-pick a2673b2a2278526079e11b5fa3b5190c9c1103b8
git cherry-pick 8a72d39b02426db12e5655d28e94edec541707e3
git cherry-pick aa7817ee280cd4420002003eee787cb1c9e7504e
git cherry-pick 1257e742e3215819bfd58498f4043f3c16dc59a8
git cherry-pick a136c208bbea66071474c95e8db02bacd31d4503
git cherry-pick 32d9e79074262b06baa47b0a6081d785e4eb592e
git cherry-pick f237a61bb478bedfd3ae8c1d4e788e5127f997a4
git cherry-pick fd14d3754f2877dcd12544b7edad583278caf695
git cherry-pick 2ebac311a71cdbd1bb73c7b40b739afe217f5b16
git cherry-pick 67c32f076534db7950ccc774d4ad61ec16b6f74f
git cherry-pick d1f2200e538b68287e28d0cb3f37022c1a129699
git cherry-pick a07a3ffc376d07108751bfe6779c21ef588890c8
git cherry-pick 2215405a54ba8daa8d751ed3391af957c8f1096d
git cherry-pick 3b26d42ac8127319b9387e6568b955461acaed87
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.7.0-rc1..HEAD --pretty='format:- %s'`

```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

- Closes #2518 RC2 Bug Fixes introduced extra space to the main navigation (#2520)
- Update drupal/inline_entity_form requirement from 1.0.0-rc15 to 2.0.0-rc4 (#2448)
- Closes #2504 News articles are not displaying thumbnail images on view displays (#2514)
- Update Arizona Bootstrap to 2.0.24 (#2522)
- Fixes #2509 Content editor role unintentionally deleted. (#2510)
- Closes #2503: Update view display for AZ Person (#2501)
- #2330 Update slack channel names from #friday-meetings to #azdigital-meetings (#2508)
- Closes #2516: Apply patch for XML sitemap D10 incompatibility issue. (#2517)
- #2331 Update slack channel names from #azdigital-general to #azdigital-support (#2507)
- Closes #2502: Update Analytics Dashboard link in toolbar. (#2511)
- Update drupal/ckeditor requirement from 1.0.1 to 1.0.2 (#2496)
- Fixes #2512 Cursor missing in new text area on Chrome (#2513)
- Closes #2495 Menu blocks appearing on pages that they should not appear on (#2500)
- Closes #2497 Reorder people row weight input boxes don't show values (#2498)
- Closes #2499: Add viewsreference patch to address Ajax failures. (#2505)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.0-rc2
